devicetypes/smartthings/zwave-switch.src/zwave-switch.groovy

```diff
@@ -156,13 +159,34 @@ def poll() {
 	])
 }
 
+
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+	refresh()
+}
+
 def refresh() {
 	delayBetween([
 		zwave.switchBinaryV1.switchBinaryGet().format(),
 		zwave.manufacturerSpecificV1.manufacturerSpecificGet().format()
 	])
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configure() {
+	unschedule()
+	schedule("0 0/5 * * * ? *", "healthPoll")
```

Comment:
I've replaced schedule() with runEvery5Minutes(). Please check and merge in dev.